model/validation.go

```diff
@@ -49,7 +49,8 @@ func IsDNS1123Label(value string) bool {
 	return len(value) <= dns1123LabelMaxLength && dns1123LabelRex.MatchString(value)
 }
 
-func validatePort(port int) error {
+// ValidatePort ...
+func ValidatePort(port int) error {
```

Comment:
We will need to export these functions to the common API repo?
I tried to normalize the code style and updated the names to be more consistent.